common/testdata/demo.rsa

```diff
@@ -0,0 +1,15 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Do our installation instructions from now on need to include instructions on setting up a public key?